VDB-1787 Add release to sentry on deploy

paytonrules opened the pull request:
I've tested this out locally - but I'm not sure how to be 100% sure it works without merging.

paytonrules committed:
Announce a release on every deploy

I tried using after_deploy, but we lose the "prod" or "staging"
environment variable.

rmulhol commented on .travis/announce.sh:
```diff
@@ -0,0 +1,41 @@
+elif [ "$ENVIRONMENT" == "staging" ]; then
+TAG=staging
+else
+   message UNKNOWN ENVIRONMENT
```
Asked about this on another PR at some point but I'm wondering if we might want to `exit 1` here? I don't think we could hit this line without an error in `.travis.yml` but at the same time it feels like it could get confusing if we did happen to make that kind of error

paytonrules replied:
I actually thought there was some esoteric reason you all didn't exit here that I didn't know :)

rmulhol reviewed (approved):
🎉

paytonrules committed:
Address PR feedback

- Go ahead and exist if environment isn't present - and use message to
highlight the errors. This is actually much cleaner.
